tasks/section1.yml

```diff
@@ -408,7 +408,7 @@
       - rule_1.1.8
 
 - name: "SCORED | 1.1.9 | AUDIT | Ensure nosuid option set on /var/tmp partition"
-  command: /bin/true
+  shell: mount | grep /var/tmp | grep nosuid
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*